config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -203,3 +203,56 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure-dualstack
+    decorate: true
+    decoration_config:
+      timeout: 2h
+    always_run: false
+    optional: true
+    path_alias: k8s.io/kubernetes
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201005-0b243c0-master
+        command:
+        - runner.sh
+        - kubetest
+        args:
+        # Generic e2e test args
+        - --test
+        - --up
+        - --down
+        - --build=quick
+        - --dump=$(ARTIFACTS)
+        # Azure-specific test args
+        - --provider=skeleton
+        - --deployment=aksengine
+        - --aksengine-agentpoolcount=2
+        - --aksengine-admin-username=azureuser
+        - --aksengine-creds=$(AZURE_CREDENTIALS)
+        - --aksengine-orchestratorRelease=1.19
+        - --aksengine-mastervmsize=Standard_DS2_v2
+        - --aksengine-agentvmsize=Standard_D4s_v3
+        - --aksengine-deploy-custom-k8s
+        - --aksengine-location=westus2
+        - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
```

Comment:
I think that this is escaping a variable, the other jobs in this repo use `$AZURE_SSH_PUBLIC_KEY_FILE`, the other format should be with braces `${AZURE_SSH_PUBLIC_KEY_FILE}`, using parenthesis is to execute what is inside

Reply:
@aojea All other azure jobs use `$(AZURE_SSH_PUBLIC_KEY_FILE)`. Where are you seeing `$AZURE_SSH_PUBLIC_KEY_FILE`?

Reply:
lol, I do not how old was the branch I was using but I can swear you the did not have it :smile: 
you are right, the rest of the repo has them this ways
TIL I honestly thought that same bash escaping rules applied here, it seems you can use parenthesis